Lombiq.HelpfulLibraries/Libraries/Localization/LocalizedHtmlStringExtensions.cs

```diff
@@ -14,7 +14,7 @@ public static class LocalizedHtmlStringExtensions
         public static IHtmlContent Json(this LocalizedHtmlString htmlString)
         {
             using var stringWriter = new StringWriter();
-            htmlString.WriteTo(stringWriter, HtmlEncoder.Default);
+            htmlString?.WriteTo(stringWriter, HtmlEncoder.Default);

```

Comment:
I think this will return an empty string when the value is `null`. It should return `null` (ie `new HtmlString("null")`) instead.